src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
I don't think project references build the `Build` target anyway. They build `GetTargetPath`, which depends on targets that effectively build enough of the project to produce what the referencing project needs. It's been a while so I may be wrong on that. But regardless, if we keep to just one metadata (`Targets`), then we can append to whatever that metadata is by default to add the targets we need for NuProj purposes.
If we think there's a scenario for needing to build a target in the referenced project but *not* collecting its `Outputs`/`Returns` items for purposes of inclusion in the package, then sure, I guess another metadata on the project reference to list the targets to include results from is useful. But let's have that be the off-case rather than the default case. And let's give that metadata a more meaningful name (e.g. PackageItemsFromTargets).